Assets/SEE/Game/Avatars/AvatarAimingSystem.cs

```diff
@@ -90,14 +90,22 @@ private void TogglePointing()
         /// <summary>
         /// If <paramref name="activate"/> is true, the laser will be turned on;
         /// otherwise turned off.
-        /// This parameter is also propagated to all clients.
+        /// This parameter is also propagated to all clients if not <paramref name="remoteRequest"/>.
         /// </summary>
         /// <param name="activate">whether pointing is to be activated</param>
-        /// <remarks>This method is called either as a interaction request of the local
+        /// <param name="remoteRequest">whether the request comes from a remote
+        /// client, in which case the change of the pointing modus will not be
```

Comment:
Shouldn't it be `mode`?
[Modus](https://en.wiktionary.org/wiki/modus)

Reply:
Yes, mode would be better. Fixed.